languages/ruby/exercises/concept/README.md

```diff
@@ -0,0 +1,27 @@
+# Ruby concept exercises
+
+The concept exercises are based on this [list of concepts][reference-shared].
+
+## Implemented exercises
+
+These are the concept exercises that have currently been implemented, as well as the concepts they teach and their prerequisite concepts:
+
+| exercise                                                            | concepts                                                            | prerequisites                                                       |
+| ------------------------------------------------------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------- |
+| [`strings`][concept-exercise-strings]                               | `strings-basic`                                                     | -                                                                   |
+
+**⚠ Note ⚠**: The idea here is to use a `concept` name for the exercise/folder, but perhaps use some sort of "progression", so they will naturally become a sort of path to traverse.
```

Comment:
I like the triangles here, but are they "safe" for general consumption?  Will it disrupt screen readers?

Reply:
I've just tried with the screen reader extension for Google Chrome and it reads `warning sign, note, warning sign` It looks good to me